build.proj

```diff
@@ -5,7 +5,10 @@
   <Target Name="Build" DependsOnTargets="PrepareOutput;InitBuild">
     <Message Text="Build Environment: $(Platform) $(Configuration) $(TargetOS) $(TargetRid)" />
 
-    <MSBuild Projects="repos\$(RootRepo).proj" Targets="Build" BuildInParallel="$(BuildInParallel)" StopOnFirstFailure="true" />
+    <PropertyGroup>
+      <PortableBuild Condition="'$(PortableBuild)' == ''">false</PortableBuild>
+    </PropertyGroup>
+    <MSBuild Projects="repos\$(RootRepo).proj" Targets="Build" BuildInParallel="$(BuildInParallel)" StopOnFirstFailure="true" Properties="PortableBuild=$(PortableBuild)" />
```

Comment:
At this point, I am probably nit picking, but most other configuration stuff (such as `Configuration` and `Platform`) gets defined in `dir.props`. Perhaps we should put `PortableBuild` there too?

Reply:
Not a nitpick, consistency is important. I've moved the default into `dir.props`, which let me drop manually passing the flag into the Project as well.